generator/src/main/java/com/github/fommil/netlib/generator/AbstractNetlibGenerator.java

```diff
@@ -201,7 +201,7 @@ protected void iterateRelevantParameters(Method method, boolean offsets, Paramet
       if (i < method.getParameterTypes().length - 1
           && param.isArray()
           && method.getParameterTypes()[i + 1] == Integer.TYPE)
-      offsetName = names[i+1];
+        offsetName = names[i+1];
```

Comment:
Right now this line is under the conditional but is at the same indentation. as the if statement and the "callback" below. Is that intended?